apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-docs.component.html

```diff
@@ -1,44 +1,48 @@
-<fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Radio Button Group with Radio buttons as Content
+<fd-docs-section-title [id]="'projection'" [componentName]="'Radio Button Group'">
+    Radio Button Group With Radio Buttons Passed as Content.
 </fd-docs-section-title>
 
-<description>This example shows Radio Button Group implementaion with Content as Radio Buttons</description>
+<description>
+    Platform Radio Buttons Passed as Content
+</description>
 <component-example>
     <fdp-platform-radio-group-content-example></fdp-platform-radio-group-content-example>
 </component-example>
 <code-example [exampleFiles]="contentRadioGroup"></code-example>
 
-<fd-docs-section-title [id]="'def'" [componentName]="'Disabled Radio Button Group'">
-    Examples for Radio Button Group with List of values
+<fd-docs-section-title [id]="'list'" [componentName]="'Disabled Radio Button Group'">
+    Radio Button Group Creates Radio Buttons With List of String Values.
 </fd-docs-section-title>
 <description>
-    This example shows Regular Radio Button Group implementaion with list of string Values.<br />
+    Creating Platform Radio Buttons with List of String Values.<br />
```

Comment:
Tagging our documentation gurus to confirm if the titles need `.` at the end
@Vanessa-Cusmich @jessmetivier

Reply:
Titles/headers of any kind definitely don't need periods :) I will need to adjust the length of the title though (later).